### ISSUE TITLE
Translate to Brazilian Portuguese

### ISSUE THREAD

**Issue by EliasPereirah** [facebookresearch/fairseq#4952]

**Comment by animaone** [comment]
Yes. This is important, the ISO specification puts portuguese from Portugal and portuguese from Brazil in the same category (por) but they are totally different languages when it comes to pronunciation. 
This issue is **URGENT**

**Comment by bryanrafaelbueno** [comment]
I hope they resolve this soon, several projects that use Hubert are limited because of this, the accent is for Portuguese, not Brazilian.
